### ISSUE TITLE
Silent installation Firebird 4.0.1 is only copying files to folder Firebird_4_0

### ISSUE THREAD

**Issue by lwennekers** [FirebirdSQL/firebird#7105]
The silent or verysilent installation by using Firebird-4.0.1.2692-0-x64.exe is not installing the service, nor the used dlls

The folowing command was used:
/SP- /VERYSILENT /COMPONENTS="ServerComponent\ClassicServerComponent,ServerComponent,DevAdminComponent,ClientComponent" /MERGETASKS="SuperClassicTask,UseServiceTask,CopyFbClientAsGds32Task"
Using this command with Firebird-4.0.0.xxxx-0-x64.exe which was functioning.

After investigation of other possibilities here I noticed some same changes and I tried:
/SP- /SILENT /COMPONENTS="ServerComponent,DevAdminComponent,ClientComponent" /TASKS="UseSuperClassicTask,UseServiceTask,CopyFbClientToSysTask,CopyFbClientAsGds32Task"
No installation of the service as well nor the dlls placed inside the correct directories.

So this problem only seams to occurs with  Firebird-4.0.1.2692-0-x64.exe

**Comment by reevespaul** [comment]
Your first example:
```
/COMPONENTS="ServerComponent\ClassicServerComponent,ServerComponent,DevAdminComponent,ClientComponent" /MERGETASKS="SuperClassicTask,UseServiceTask,CopyFbClientAsGds32Task"
```
has two errors, as you have discovered. There is no 'ServerComponent\ClassicServerComponent' and neither is there a 'SuperClassicTask'. If it worked you were just lucky, I guess.

Your second example uses the /COMPONENTS parameter. This _ought_ to work. But it looks as if it doesn't. That looks like it is a bug in the InnoSetup installer. In any case, you are just specifying the defaults so the /COMPONENTS parameter is not needed.

When I execute this command:
```
Desktop\Firebird-4.0.1.2692_0_x64.exe  /SP- /VERYSILENT /TASKS="UseSuperClassicTask,UseServiceTask,CopyFbClientAsGds32Task,CopyFbClienttoSysTask"
```
Firebird installs the tasks specified correctly.

I don't think there is anything to be done here. I did think about updating the scripted install readme, but that seems to be more or less up to date:

  NOTE - If a full server instal is required there is no need to 
  specify /COMPONENTS. All three are chosen by default.

Although I will fix the spelling and add a note that specifying the default components seems to be broken.

**Comment by reevespaul** [comment]
That will be fixed as part of #7096
